### DISCUSSION TITLE
Permission denied while opening a bagfile on internal HDD

### DISCUSSION THREAD

**Discussion by botmayank** [foxglove/community#692] [Q&A]
When I try to open a bagfile on my Ubuntu 20.04 x86 machine residing on the internal HDD of my laptop, I get a permission denied pop up with the heading "Could not read the contents of <dir_name>: Error opening <dir_path>: Permission denied" even though the disk is properly R/W mounted and I can open the bag in `rqt_bag`. How can I debug this further? For large bags I don't have space in my internal SSD to always move them to open.

I'm on Foxglove Studio 1.49.0 

Love the tool, would be great to get help with this.

**Comment by botmayank** [comment]
Permission for the folder is 0o777 and group and owner is $USER. Problem persists with `foxglove-studio` 1.50.0

**Comment by botmayank** [comment]
I'm able to play the bagfile in cmdline and open it up in foxglove by connecting to the ROS1 master locally, just not able to add it as a data source directly.

**Reply by jtbandes** [reply]
Can you open up the developer console (press alt > view menu > advanced) and see if there are any more detailed error messages?
Can you also try on the latest version, and on studio.foxglove.dev?

**Reply by botmayank** [reply]
There's no error in the Console since it's from the OS layer itself, not the web browser application part of it.

**Comment by defunctzombie** [comment]
Which installation method did you use for the desktop app?

**Reply by botmayank** [reply]
I installed using snap
